WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginListActivity.java

```diff
@@ -102,16 +105,16 @@ public void onSaveInstanceState(Bundle outState) {
     }
 
     private void setupViews() {
-        RecyclerView recyclerView = (RecyclerView) findViewById(R.id.plugins_recycler_view);
-        recyclerView.setItemAnimator(new DefaultItemAnimator());
-        recyclerView.setLayoutManager(new LinearLayoutManager(this));
-        DividerItemDecoration dividerItemDecoration = new DividerItemDecoration(recyclerView.getContext(),
+        mRecyclerView = findViewById(R.id.plugins_recycler_view);
```

Comment:
I love that we don't need to cast these anymore!

Reply:
Me too!

Although now we have a bunch of code that still casts them and AS is complaining about it. It's your fault AS!